swagger.yaml

```diff
@@ -3678,6 +3678,10 @@ definitions:
         x-go-custom-tag: gorm:"type:text"
         type: string
         description: Array of image statuses.
+      api_domain_name_resolution:
```

Comment:
Name should be domain_name_resolution because it contains all resolutions, not just the API resolution

Reply:
domain_name_resolutions

Reply:
done, was changed to domain_name_resolutions